modules/ossm-installing-multi-primary-multi-network-kiali.adoc

```diff
@@ -0,0 +1,189 @@
+// This procedure is used in the following assembly:
+// * install/ossm-multi-cluster-topologies.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-installing-multi-primary-multi-network-kiali_{context}"]
+= Installing a multi-primary multi-network mesh by using Kiali
+
+Install {istio} in the multi-primary multi-network topology on two {ocp-product-title} clusters.
+
+[NOTE]
+====
+In this procedure, `CLUSTER1` is the East cluster and `CLUSTER2` is the West cluster.
+====
+
+You can adapt these instructions for a mesh spanning more than two clusters.
+
+.Prerequisites
+
+* You have installed the latest Kiali Operator on each cluster.
+
+* Istio installed in a multi-cluster configuration on each cluster.
+
+* You have `istioctl` installed on the laptop you can use to run these instructions.
+
+* You are logged in to the {ocp-product-title} web console as a user with the `cluster-admin` role.
+
+* Kiali has a single endpoint to query metrics and a single endpoint to query traces where aggregated data across all clusters resides. There are multiple ways to aggregate the data, such as federating Prometheus or using OTEL collector pipelines.
+
+.Procedure
+
+. Install Kiali on the East cluster:
+
+.. Create a YAML file named `kiali.yaml` that creates a namespace for the Kiali deployment. 
++
+.Example configuration
+[source,yaml,subs="attributes,verbatim"]
+----
+apiVersion: kiali.io/v1alpha1
+kind: Kiali
+metadata:
+  name: kiali
+  namespace: istio-system
+spec:
+  version: default
+----
+
+.. Apply the YAML file on the East cluster by running the following command:
++
+[source,terminal]
+----
+$ oc --context east apply -f kiali.yaml
```

Comment:
Prerequisite instructions say `CLUSTER1` and `CLUSTER2` are the names of the clusters but then `east` is used here as the context. It should probably be `cluster1` and `cluster2` to be consistent with the OSSM multi-cluster instructions.